telcell/celldb/pgdatabase.py

```diff
@@ -41,6 +41,9 @@ def _build_antenna(row: Tuple) -> Properties:
 
 
 def _build_cell_identity_query(ci):
+    if ci is not None and not isinstance(ci, CellIdentity):
```

Comment:
`isinstance` can handle `None`, will be false as well.

```suggestion
    if not isinstance(ci, CellIdentity):
```